R/loadData.R

```diff
@@ -0,0 +1,29 @@
+#' These functions are to pre-load data for some cities so they can be easily visualized
+
+# how do i access functions in other files? should this stuff be in data-raw.R or smthin?
```

Comment:
the functions are in our package, so if you library(censusviz) at the top, you can access them

Reply:
This file should be in data-raw